src/yadmark/benchmark/external/apfelpy_utils.py

```diff
@@ -0,0 +1,231 @@
+"""
+    Utilities to help run APFEL++ benchmarks.
+"""
+import numpy as np
+
+# Q2 knots specs
+NQ = 50
+QMin = 1
+QMax = 200
+
+# Map observables names to APFEL++ methods
+MAP_HEAVINESS = {
+    "charm": 4,
+    "bottom": 5,
+    "top": 6,
+    "light": 0, # TODO: Check combination
```

Comment:
Here we'd want to also allow for a compound matching given how the light contributions are computed in Apfel++:

https://github.com/vbertone/apfelxx/blob/4ebe6952888f238457fb0b72cdfec5963b26b553/pywrap/structurefunction_nc_test.py#L51.

Reply:
yes regarding this if summing only on 1,2,3 is easy enough we can do it otherwise we look only at total. 
In the end we can't accommodate all the definitions which are around. We know polarised is not making any difference in this respect (ie heaviness)

Reply:
@giacomomagni I think this has now been more or less fixed. What I don't fully understand though is why these `pids` have to be passed to the coupling functions? In some of Valerio's examples, this entry is not needed.

https://github.com/NNPDF/yadism/blob/f64423182627df92e4035d7e7a31316b97fa5968/src/yadmark/benchmark/external/apfelpy_utils.py#L42-L46

Reply:
I think this function returns a list of couplings for each pid. If not given all couplings are returned, no?

Reply:
But to be consistent with Valerio's examples, should we remove this entry?

Reply:
> fine, I think in Valerio's examples he's always computing total that's why he is not specifying any pid.

Wait, he is also computing [light, charm, bottom structure functions](https://github.com/vbertone/apfelxx/blob/00238cd9d5711e6ac16e88dad1e17ba08761c32b/pywrap/structurefunction_nc_pol_test.py#L50-L53).

Reply:
I also think the light combination will not match due to the definition. If I recall correct we don't match apfel light definition ( @felixhekhorn is this correct? ), so most likely we don't also here.

Reply:
> Wait, he is also computing [light, charm, bottom structure functions](https://github.com/vbertone/apfelxx/blob/00238cd9d5711e6ac16e88dad1e17ba08761c32b/pywrap/structurefunction_nc_pol_test.py#L50-L53).

But he is always doing total and the other... Never only charm or bottom

Reply:
"light" is in a complex state, because I believe its meaning might depend on the FNS (I should check). Originally, we were defining "light" as zero-mass, but this would not work for FONLL, so I'm almost sure "light" in FONLL means another thing.

However, this hopefully will simplify after the implementation of "numerical FONLL", that for yadism means essentially drop FONLL implementation, and implement just the ingredients.

Reply:
Indeed I believe we can just drop light here, 
as said we haven't changed anything in that respect with polarised.